src/grimp/application/ports/graph.py

```diff
@@ -284,13 +284,15 @@ def find_illegal_dependencies_for_layers(
 
         Additionally, multiple layers can be grouped together at the same level; for example
         `mypackage.utils` and `mypackage.logging` might sit at the bottom, so they cannot
-        import from any other layers. Layers at the same level must be independent, so any
-        dependencies in either direction will be treated as illegal.
+        import from any other layers. When a simple set of sibling layers is passed then they
```

Comment:
Would slightly prefer the documentation (at least in this docstring) to indicate `Level` as being the primary type that is passed, and then mention the single-string / set of strings as conveniences that will be converted to levels. But not a blocker.